client/src/util/stateManager/viewStackHelpers.ts

```diff
@@ -109,6 +109,14 @@ export function _userResetSubsetAnnoMatrix(
   return annoMatrix;
 }
 
+const isAnnoMatrixClipView = (
```

Comment:
did you mean to add this file change to this PR? I might be missing something lol

Reply:
I was just peering into this file while reading your debugging findings and thought typing it would be helpful, lol. Not sure when else i would commit this